src/popup/router/pages/SuccessTip.vue

```diff
@@ -77,18 +77,12 @@ export default {
     },
   },
   async created() {
-    try {
-      this.verifiedUrls = (await axios.get(`${BACKEND_URL}/verified`)).data;
-    } catch (e) {
-      this.$logError({ e, action: 'fetch-verified' });
-    }
+    this.verifiedUrls = (
+      await axios.get(`${BACKEND_URL}/verified`).catch(() => ({ data: [] }))
+    ).data;
     const { addresses, tab } = await this.$store.dispatch('getWebPageAddresses');
     if (addresses.length) {
-      try {
-        await axios.post(`${TIP_SERVICE}`, { url: tab.url, address: addresses[0] });
-      } catch (e) {
-        this.$logError({ e, url: tab.url, action: 'claim' });
```

Comment:
I assume that it is not necessary to handle error here because of the global axios handler in `src/lib/request.js`.

Reply:
I think `src/lib/request.js` currently is not used. My idea was to to replace `axios.` with `request.` and this way use the error handler but I was not sure if we can extract the action name this way, so I left it